db/migrate/0036_create_ronin_web_vulns_table.rb

```diff
@@ -50,7 +50,7 @@ def change
       t.boolean :sqli_escape_parens, null: true
       t.boolean :sqli_terminate, null: true
 
-      t.index :type, unique: true
+      t.index [:type, :query_param, :header_name, :cookie_param, :form_param, :url], unique: true, name: 'vuln_index'
```

Comment:
Is `vuln_index` ok for index name?

Reply:
Probably name it `index_ronin_url_web_vulns_unique` to match the other index names.